Formula/grpc.rb

```diff
@@ -32,9 +31,23 @@ class Grpc < Formula
   depends_on "protobuf"
   depends_on "re2"
 
+  uses_from_macos "llvm" => [:build, :test], since: :catalina
```

Comment:
Do we actually need it at test time?

I also don't think this should apply to Linux.